src/sentry/tasks/post_process.py

```diff
@@ -137,9 +136,8 @@ def post_process_group(event, is_new, is_regression, is_new_group_environment, *
         from sentry.rules.processor import RuleProcessor
         from sentry.tasks.servicehooks import process_service_hook
 
-        # Re-bind node data to avoid renormalization. We only want to
-        # renormalize when loading old data from the database.
-        event.data = EventDict(event.data, skip_renormalization=True)
+        # Clear node id to prevent reloading data from nodestore in post process
+        event.clear_node_id()
```

Comment:
This definitely needs be double checked by someone, but this is the part of re-binding node data that I believe is important since clearing the ID prevents any further nodestore access or attempts to rebind data.